test/integration-test/src/tests/load.rs

```diff
@@ -165,4 +198,45 @@ fn pin_lifecycle() {
 
     // program should be unloaded
     assert_loaded!("pass", false);
+
+    // Delete veth interface pait
+    delete_veth_pair(TEST_VETH_NAME).unwrap();
+}
+
+fn create_veth_pair(veth_name: &str) -> Result<(), rtnetlink::Error> {
+    let (connection, handle, _) = new_connection().unwrap();
+    tokio::spawn(connection);
+    block_on(async {
```

Comment:
Since we enabled tokio in the whole integration-tests crate, you don't need this `block_on`. Just make the whole function async.

Reply:
then I'll have to make the test functions also async...is that what you want? or are you suggesting something else?